cdap-docs/application-templates/source/etl/templates/transformations/validator.rst

```diff
@@ -0,0 +1,47 @@
+.. meta::
+:author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+==========================
+Transformations: Validator
+==========================
+
+.. rubric:: Description
+
+A transform that validates records using a custom Javascript function based on a set of available validator functions.
+
+.. rubric:: Use Case
+
+The transform is used when you need to validate records.
+For example, you may want to validate records that are valid IP address or valid dates and
+log error's if they aren't valid.
+
+.. rubric:: Properties
+
+**validators** Comma separated list of validators that are used by the validationScript. Example: "validators" : "core"
```

Comment:
Example: "validators" : "core"
that's not a comma-separated list...what would be a better example?